docs/source/dataframe-design.rst

```diff
@@ -111,12 +111,33 @@ scans).
 .. code-block:: python
 
     >>> col_known = ddf.col.cat.as_known()  # use for single column
+    >>> col_known.cat.known
+    True
     >>> ddf_known = ddf.categorize()        # use for multiple columns
+    >>> ddf_known.col.cat.known
+    True
 
 To convert a known categorical to an unknown categorical, there is also the
 ``.cat.as_unknown()`` method. This requires no computation, as it's just a
 change in the metadata.
 
+Non-categorical columns can be converted to categoricals in a few different
+ways:
+
+- ``ddf.astype({col_name: 'category', ...})`` and
+  ``ddf.col.astype('category')`` operate lazily, and result in unknown
```

Comment:
Might also consider putting these in a code block with comments rather than a bulleted list